msteams-platform/apps-in-teams-meetings/teams-live-share-canvas.md

```diff
@@ -0,0 +1,313 @@
+
+You can customize this behavior in the following ways:
+
+- Changing the starting reference point to the top-left corner of the canvas.
+- Alter the offset x and y positions of the viewport.
+- Change the scale level of the viewport.
+
+> [!NOTE]
+> Reference points, offsets, and scale levels are local to the client and are not synchronized across meeting participants.
+
+Example:
+
+```html
+<body>
+  <button id="pan-left">Pan left</button>
+  <button id="pan-up">Pan up</button>
+  <button id="pan-right">Pan right</button>
+  <button id="pan-down">Pan down</button>
+  <button id="zoom-out">Zoom out</button>
+  <button id="zoom-in">Zoom in</button>
+  <button id="change-reference">Change reference</button>
+</body>
+```
+
+```javascript
+// ...
+
+// Pan left
+document.getElementById("pan-left").onclick = () => {
+  inkingManager.offset = {
+    x: inkingManager.offset.x - 10,
+    y: inkingManager.offset.y,
+  };
+};
+// Pan up
+document.getElementById("pan-up").onclick = () => {
+  inkingManager.offset = {
+    x: inkingManager.offset.x,
+    y: inkingManager.offset.y - 10,
+  };
+};
+// Pan right
+document.getElementById("pan-right").onclick = () => {
+  inkingManager.offset = {
+    x: inkingManager.offset.x + 10,
+    y: inkingManager.offset.y,
+  };
+};
+// Pan down
+document.getElementById("pan-down").onclick = () => {
+  inkingManager.offset = {
+    x: inkingManager.offset.x,
+    y: inkingManager.offset.y + 10,
+  };
+};
+// Zoom out
+document.getElementById("zoom-out").onclick = () => {
+  if (inkingManager.scale > 0.1) {
+    inkingManager.scale -= 0.1;
+  }
+};
+// Zoom in
+document.getElementById("zoom-in").onclick = () => {
+  inkingManager.scale += 0.1;
+};
+// Change reference
+document.getElementById("change-reference").onclick = () => {
+  if (inkingManager.referencePoint === "center") {
+    inkingManager.referencePoint = "topLeft";
+  } else {
+    inkingManager.referencePoint = "center";
+  }
+};
+```
+
+## Ideal scenarios
+
+With web pages coming in all shapes and sizes, it isn't possible to make Live Share canvas work for every scenario. The package is ideal for scenarios at which all users are looking at the same content at the same time. While not all of the content needs to be visible on the screen, it must be content that scales across devices linearly.
+
+Here are a couple examples of scenarios where Live Share canvas is a great option for your application:
+
+- Overlay images and videos that render with the same aspect ratio on all clients.
+- Viewing a map, 3D model, or whiteboard from the same rotation angle.
+
+The reason both of these scenarios work well is because being viewed is the same on all devices, even though users may be looking at it with different zoom levels and offsets. If your app content doesn't fit one of these scenarios, it likely isn't a good fit for your scenario.
```

Comment:
You seem to have something missing between "because" and "being viewed".

I would rephrase the second sentence into "If your app's layout and content change depending on screen size, and it is not possible or practical to generate a temporary reference/common view for all participants, Live Share canvas might not be a good fit for your scenario."